scripts/ci/travis-install.sh

```diff
@@ -35,7 +35,7 @@ else
   ./scripts/build/check-dependency.sh make
   ./scripts/build/check-dependency.sh npm
 
-  npm install -g npm@3.10.10
+  npm install -g npm@3.10.10 uglify-es@3.0.3
```

Comment:
E.g. for `pip` python packages we can use https://pip.readthedocs.io/en/1.1/requirements.html